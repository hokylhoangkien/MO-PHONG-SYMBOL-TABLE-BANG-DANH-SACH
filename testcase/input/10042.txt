INSERT k string
INSERT m string
BEGIN
INSERT k string
INSERT m string
RPRINT
END