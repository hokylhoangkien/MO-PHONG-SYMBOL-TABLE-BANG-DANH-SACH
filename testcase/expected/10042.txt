success
success
success
success
m//1 k//1